File: game/src/main/java/com/matag/game/security/SecurityToken.java
package com.matag.game.security;

import lombok.Value;

@Value
public class SecurityToken {
  private final String sessionId;
  private final String adminToken;
  private final String gameId;

  @Override
  public String toString() {
    return "SecurityToken{" +
      "sessionId='********'" +
      ", adminToken='********'" +
      ", gameId='" + gameId + '\'' +
      '}';
  }
}

File: src/main/kotlin/com/matag/game/security/SecurityToken.kt
package com.matag.game.security

data class SecurityToken(
    val sessionId: String,
    val adminToken: String,
    val gameId: String
) {
    override fun toString(): String {
        return "SecurityToken{sessionId='********', adminToken='********', gameId='$gameId'}"
    }
}
